test_validator.py

```diff
@@ -4,7 +4,17 @@
 
 
 class TestValidator(unittest.TestCase):
-    def setUp(self):
+
+    def test_validate_empty_rules(self):
+        self.validator = Validator()
+        self.assertEqual(True, self.validator.validate({}))

```

Comment:
Why here the validation must return True and on  test_validate_data_empty it returns False

Reply:
I am trying to check an empty validator.
When the validator has no rules to validate, any data (empty or not empty) should be valid.
The correct behavior example is on line 12